File: Python/simp_calc.py
a=int(input("Enter any number: "))
b=int(input("Enter another number:"))
print("")
print("\t****MENU****")
print("1) Addition\n2) Substraction\n3) Multiplication\n4) Division")
print("")
op=int(input("Enter your choice: "))

while op!='5':   
    if op=='1':
            c=a+b
            print(a,"+",b,"=",c)
    elif op=='2':
        c=a+b
        print(a,"-",b,"=",c)
    elif op=='3':
        c=a+b
        print(a,"*",b,"=",c)
    elif op=='4':
        c=a+b
        print(a,"/",b,"=",c)
    else:
        print("Wrong Choice!!!")
        op=5
print("Exiting!!!")

File: C++/simp_calc.cpp
# include <iostream>
using namespace std;

int main() {

  int op;
  float n1, n2;

    do{
      cout << "Enter two operands: ";
      cin >> n1 >> n2;

      cout << "\t*MENU*\n\t1) Addition\n\t2) Substraction\n\t3) Multiplication\n\t4) Division\n\t5) Exit\nEnter your choice: ";
      cin >> op;

      switch(op) {

        case 1:
          cout << n1 << " + " << n2 << " = " << n1 + n2 << endl;
          break;

        case 2:
          cout << n1 << " - " << n2 << " = " << n1 - n2 << endl;
          break;

        case 3:
          cout << n1 << " * " << n2 << " = " << n1 * n2 << endl;
          break;

        case 4:
          cout << n1 << " / " << n2 << " = " << n1 / n2 << endl;
          break;

        case 5:
          cout << " Coming out!!! " << endl;
          break;

        default:
          cout << "Incorrect choice!";
          break;
      }
    }while(op!=5);

  return 0;
}
